storage/storage-transfer/README.md

```diff
@@ -54,18 +54,21 @@ Creating a one-time transfer from Amazon S3 to Google Cloud Storage.
   1. Set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
 1. In AwsRequester.java, fill in the user-provided constants.
 1. Compile the package with
-   ```
-   mvn compile
-   ```
+
```

Comment:
This, unfortunately, resets the numbering. We should either use bulletted lists, or indent the heck out of this block in hopes that it'll format correctly :-)